fixtures/chauvet-professional/colorado-1-solo.json

```diff
@@ -380,44 +585,60 @@
         },
         {
           "dmxRange": [10, 29],
-          "type": "Speed",
-          "speed": "stop",
+          "type": "Maintenance",
           "comment": "Linear Dimmer"
         },
         {
           "dmxRange": [30, 69],
-          "type": "Speed",
-          "speed": "fast",
+          "type": "Maintenance",
           "comment": "NonLinear Dimmer Curve 1 (Fastest)"
         },
         {
           "dmxRange": [70, 129],
-          "type": "Speed",
-          "speed": "fast",
+          "type": "Maintenance",
           "comment": "NonLinear Dimmer Curve 2"
         },
         {
           "dmxRange": [130, 189],
-          "type": "Speed",
-          "speed": "slow",
+          "type": "Maintenance",
           "comment": "NonLinear Dimmer Curve 3"
         },
         {
           "dmxRange": [190, 255],
-          "type": "Speed",
-          "speed": "slow",
+          "type": "Maintenance",
           "comment": "NonLinear Dimmer Curve 4 (Slowest)"
         }
       ]
+    },
+    "Hue": {
+      "fineChannelAliases": ["Hue fine"],
+      "capability": {
+        "type": "Generic"
+      }
+    },
+    "Saturation": {
+      "capability": {
+        "type": "Generic"
+      }
+    },
+    "Step Duration": {
+      "capability": {
+        "type": "EffectDuration"
+      }
+    },
+    "Fade Duration": {
+      "capability": {
+        "type": "EffectDuration"
+      }
```

Comment:
`EffectDuration` capability type needs a specific `duration` (or probably rather `durationStart` and `durationEnd`)

Reply:
Makes sense. Maybe add a `helpWanted` message so it can be checked?